File: BenchProject1/Models/Company.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace BenchProject1.Models
{
    public class Company
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public string Id { get; set; }
        public string Name { get; set; }
        public string Code{ get; set; } 
        public string Description { get; set; }
    }
}

File: ClientApp/bench/src/app/models/company.ts
import { Component, Injectable, Input } from "@angular/core";

export class Company{    

    name: string;
    code: string;
    description: string;

    constructor(_name: string, _code: string, _description: string) {
        this.name = _name;
        this.code = _code;
        this.description = _description;        
    }
}